git-hooks.nix

```diff
@@ -0,0 +1,12 @@
+{ inputs, ... }:
+{
+  perSystem = { system, ... }:
+    {
+      checks.pre-commit-check = inputs.pre-commit-hooks.lib.${system}.run {
```

Comment:
You don't have to manually create this check, because upstream repo provides its own flake-parts module that we can import.

Reply:
@srid you mean [this](https://flake.parts/options/pre-commit-hooks-nix)?

Reply:
Yes.